doc/nrf/libraries/bin/lwm2m_carrier/CHANGELOG.rst

```diff
@@ -14,6 +14,68 @@ Certification status
 
 For certification status of the released versions, see `Mobile network operator certifications`_.
 
+liblwm2m_carrier 3.5.0
+**********************
+
+Release for modem firmware version 1.3.6 and 2.0.1.
+
+Size
+====
+
+See :ref:`lwm2m_lib_size` for an explanation of the library size in different scenarios.
+
++-------------------------+---------------+------------+
+|                         | Flash (Bytes) | RAM (Bytes)|
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (binary)                |               |            |
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (reference application) |               |            |
++-------------------------+---------------+------------+
+
+Changes
+=======
+
+* Added the :kconfig:option:`LWM2M_CARRIER_SOFTBANK_DIVIDED_FOTA` which allows the carrier to handle the proprietary SoftBank FOTA images.
+
+  * The glue layer now handles the image type :c:macro:`LWM2M_OS_DFU_IMG_TYPE_APPLICATION_FILE which allows an update to be split into several files.
+
+* Added a callback for the Modem functional mode changes (AT CFUN). For more information see :ref:`mlil_callbacks`.
+
+  * This allows the LwM2M carrier library to resume AT notification subscriptions after the application powers off an on the modem.
+
+* Changed the ``server_binding`` parameter to use :c:macro:`LWM2M_CARRIER_SERVER_BINDING_UDP` or :c:macro:`LWM2M_CARRIER_SERVER_BINDING_NONIP` (instead of ``U`` or ``N``).
+
+* Changed the Lwm2M Carrier Library Kconfig menu to clarify some the option dependencies.
```

Comment:
```suggestion
* Changed the ``server_binding`` parameter to use :c:macro:`LWM2M_CARRIER_SERVER_BINDING_UDP` or :c:macro:`LWM2M_CARRIER_SERVER_BINDING_NONIP` (instead of ``U`` or ``N``).
* Changed the Lwm2M carrier Library Kconfig menu to clarify some the option dependencies.
```